platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/geometry/LatLngBoundsTest.java

```diff
@@ -194,6 +211,66 @@ public void testOuterUnion() {
   }
 
   @Test
+  public void northWest() {
```

Comment:
@tobrun For all these new methods, could we add specific tests for cases where latitude and/or longitude are crossing their limits (i.e International Date Line)?